x-pack/test/functional/apps/lens/group3/open_in_lens/agg_based/metric.ts

```diff
@@ -48,5 +55,168 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         },
       ]);
     });
+
+    it('should convert aggregation with params', async () => {
+      await visEditor.clickMetricEditor();
+      await visEditor.selectAggregation('Average', 'metrics');
+      await visEditor.selectField('machine.ram', 'metrics');
+      await visEditor.clickGo();
+
+      const button = await testSubjects.find('visualizeEditInLensButton');
```

Comment:
Also noted that in @VladLasitsa s PR - this seems to be a super common thing, could we put it into the page object?

Reply:
Done.